src/core/tactics.ml

```diff
@@ -109,6 +109,8 @@ let handle_tactic : Sig_state.t -> Proof.t -> p_tactic -> Proof.t =
       handle_refine (Rewrite.symmetry ss tac.pos ps)
   | P_tac_why3(config)  ->
       handle_refine (Why3_tactic.handle ss tac.pos config g)
+  | P_tac_fail          ->
+      fatal tac.pos "Fail tactic"
```

Comment:
As @francoisthire suggested, it would be a good idea to take time to rethink our error messages at some point. Here, I'd rather say `Intended failure`, which, IMO seems a bit friendlier and less debug-ish, but it's really subjective. What do you think?